src/styles/Theme.tsx

```diff
@@ -16,8 +16,8 @@ const theme: DefaultTheme = {
     main: 'Inter',
     fontSize: {
       body: {
-        min: '20px',
-        size: '24px',
+        min: '18px',
+        size: '18px',
```

Comment:
size should be relative value instead of absolute!

Reply:
for themes, relative values does not seem to change the font size